pykrige/ok.py

```diff
@@ -705,6 +714,12 @@ def execute(self, style, xpoints, ypoints, mask=None, backend='vectorized',
                                         np.sin(lon_p) * np.cos(lat_p),
                                         np.sin(lat_p)), axis=1)
 
+            # Packed-complex version. 

```

Comment:
Regarding complex packing: I think there was a mapping function or technique that required a signature with a single argument. This may not be an issue any more.